bin/deploy-hobby

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
```

Comment:
I know we don't have a CI check for this (yet) but can you run this via https://github.com/koalaman/shellcheck ? There are few warnings that I think we should fix

Reply:
done!